source/linux/Foundational_Components/Kernel/Kernel_Drivers/E5010_JPEG_Encoder.rst

```diff
@@ -172,22 +174,44 @@ Due to runtime power management feature, when JPEG encoder
 is not being used by any of the applications, it stays in suspended
 state and same can be verified using k3conf utility as shown below :
 
-.. code-block:: console
+.. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+   .. code-block:: console
+
+    root@am62axx-evm:~#  k3conf dump device 201
+    |------------------------------------------------------------------------------|
+    | VERSION INFO                                                                 |
+    |------------------------------------------------------------------------------|
+    | K3CONF | (version 0.3-nogit built Thu Jul 25 14:13:02 UTC 2024)              |
+    | SoC    | AM62Ax SR1.0                                                        |
+    | SYSFW  | ABI: 4.0 (firmware version 0x000a '10.0.8--v10.00.08 (Fiery Fox))') |
+    |------------------------------------------------------------------------------|
+
+    |---------------------------------------------------|
+    | Device ID | Device Name        | Device Status    |
+    |---------------------------------------------------|
+    |   201     | AM62AX_DEV_JPGENC0 | DEVICE_STATE_OFF |
+    |---------------------------------------------------|
+
+.. ifconfig:: CONFIG_part_variant in ('J722S', 'AM67')
+
+   .. code-block:: console
 
-   root@am62axx-evm:~#  k3conf dump device 201
-   |------------------------------------------------------------------------------|
-   | VERSION INFO                                                                 |
-   |------------------------------------------------------------------------------|
-   | K3CONF | (version 0.3-nogit built Thu Jul 25 14:13:02 UTC 2024)              |
-   | SoC    | AM62Ax SR1.0                                                        |
-   | SYSFW  | ABI: 4.0 (firmware version 0x000a '10.0.8--v10.00.08 (Fiery Fox))') |
-   |------------------------------------------------------------------------------|
-
-   |---------------------------------------------------|
-   | Device ID | Device Name        | Device Status    |
-   |---------------------------------------------------|
-   |   201     | AM62AX_DEV_JPGENC0 | DEVICE_STATE_OFF |
-   |---------------------------------------------------|
+    root@j722s-evm-evm:~#  k3conf dump device 201
```

Comment:
The content in these code-blocks needs to be indented with 6 spaces

Reply:
Done.